<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>141</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ARK Damage Calculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_input">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>INPUT</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_Weapon">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::WinPanel</enum>
            </property>
            <property name="text">
             <string>Weapon: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_Weapon">
            <item>
             <property name="text">
              <string>Stone Pick</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Stone Hatchet</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Torch</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Metal Pick</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Metal Hatchet</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Metal Sickle</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Pike</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Wooden Club</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Metal Sword</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Electric Prod</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Slingshot</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Wooden Bow</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Crossbow</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Compound Bow</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Simple Pistol</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Longneck Rifle</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Shotgun</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Fabricated Pistol</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Pump-Action Shotgun</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Assault Rifle</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Fabricated Sniper Rifle</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Rocket Launcher</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_damagepercent">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::WinPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="text">
             <string>Damage %</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_damagepercent">
            <property name="minimum">
             <double>100.000000000000000</double>
            </property>
            <property name="maximum">
             <double>650.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_output">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>OUTPUT</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Raw Damage: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumber_rawdamage">
            <property name="digitCount">
             <number>6</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>CALCULATE!</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>389</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
